resources/views/volumes/clone.blade.php

```diff
@@ -47,18 +47,17 @@
                                    placeholder="Select destination project"
                                    title="Select project to clone volume to"
                                    v-on:select="setProject" :clear-on-select="false"
-                                   :value="setDefaultProject()" required></typeahead>
+                                   :value="defaultProjectName" required></typeahead>
                     </div>
 
-                    <div class="checkbox" v-cloak>
+                    <div class="checkbox">
                         <label><input type="checkbox" id="files" v-model="filterFiles">
-                            <span v-text="'Filter '+getFileType(false)+'s'"></span>
-                            <div class="form-group{{ $errors->has('clone_files') ? ' has-error' : '' }}">
+                            <span>Filter files</span>
+                            <div class="form-group{{ $errors->has('clone_files') ? ' has-error' : '' }}" v-cloak>
```

Comment:
Only use `v-cloak` on elements with `v-if` :wink: So this should be added to the `<span v-if="filterFiles">` help block below. Otherwise, the default help text will be flashing on page load.

Also in Bootstrap 3, a checkbox does not need a `form-group`. Instead, it is wrapped in a `has-error` (see [here](https://getbootstrap.com/docs/3.3/css/#forms-control-validation)).